app/templates/views/guidance/edit-and-format-messages.html

```diff
@@ -96,4 +97,20 @@ <h2 class="heading-medium" id="optional-content">Add optional content</h2>
 
   <p class="govuk-body">If you upload a list, the column names need to match the optional content in your template.</p>
 
+  <p class="govuk-body">You cannot use personalisation within optional content.</p>
+
+  <h2 class="heading-medium" id="api-formatting">Formatting when using our API</h2>
+
+  <p class="govuk-body">Placeholders can contain all the usual Notify content formatting, except for:</p>
+
+  <ul class="list list-bullet">
+    <li>line breaks</li>
+    <li>bullet points</li>
+  </ul>
+
+  <p class="govuk-body">To create a line break, use the new line character \n in the placeholder.</p>
+
+  <p class="govuk-body">To create a list of bullet points, set the placeholder value to be a list of strings. Each string will become a new bullet point. For example: placeholder_name = ["my first bullet", "my second bullet", "my third bullet"]</p>
```

Comment:
```suggestion
  <p class="govuk-body">To create a list of bullet points, set the placeholder value to be a list of strings. Each string will become a new bullet point. For example: ["my first bullet", "my second bullet", "my third bullet"]</p>
```

Using `placeholder_name =` may be confusing as it looks more like an assignment statement in code than part of a personalisation payload. I think it's still pretty clear with just the value itself.